### PR TITLE
ui: workflow description as markdown

### PR DESCRIPTION
**Pull request by safchain**

### CODE REVIEW

**Comment by safchain** [comment]
run skydive-scale-tests
run skydive-k8s-tests

**Comment by safchain** [comment]
run skydive-functional-tests-backend-elasticsearch
run skydive-k8s-tests

File: statics/js/components/workflow.js [hunk 1]
```diff
@@ -99,6 +96,26 @@ Vue.component('workflow-params', {
 
   methods: {
 
+    beautifyDesc: function(out) {
+      var i = 0
+      var href = function() {
+        i++;
+        return '<h1 onclick="$(\'#wf-arrow-' + i + '\').toggleClass(\'down\')" \
```
Comment (masco-enovance) [inline statics/js/components/workflow.js]: tested, it works fine!
I feel the title looks big like the main one, how about reducing the size to ```<h5>``` or ```<h4>```
just my suggestion ;-)

Comment (safchain) [reply statics/js/components/workflow.js]: I can reduce a bit the size of all h* tags of markdown conversion

**Comment by safchain** [comment]
run skydive-functional-tests-backend-elasticsearch
run skydive-k8s-tests

**Comment by masco-enovance** [comment]
run skydive-functional-tests-backend-elasticsearch

**Comment by safchain** [comment]
run skydive-functional-tests-backend-elasticsearch

**Comment by safchain** [comment]
run skydive-functional-tests-backend-elasticsearch

**Comment by safchain** [comment]
run skydive-functional-tests-backend-elasticsearch

**Comment by safchain** [comment]
run skydive-functional-tests-backend-elasticsearch

**Pull request merged**